Makefile

```diff
@@ -497,17 +513,16 @@ finish: startup_files libc
 	    | grep -v '^#define __FLT16_' \
 	    > "$(SYSROOT_SHARE)/predefined-macros.txt"
 
-	#
-	# The build succeeded! The generated sysroot is in $(SYSROOT).
-	#
-
-check: finish
 	# Check that the computed metadata matches the expected metadata.
 	# This ignores whitespace because on Windows the output has CRLF line endings.
 	diff -wur "$(CURDIR)/expected/$(MULTIARCH_TRIPLE)" "$(SYSROOT_SHARE)"
 
+	#
+	# The build succeeded! The generated sysroot is in $(SYSROOT).
+	#
+
 install: finish
 	mkdir -p "$(INSTALL_DIR)"
 	cp -r "$(SYSROOT)/lib" "$(SYSROOT)/share" "$(SYSROOT)/include" "$(INSTALL_DIR)"
 
-.PHONY: default startup_files libc finish check install include_dirs
+.PHONY: default startup_files libc finish install include_dirs
```

Comment:
Don't you feel like there is value is having a separate check rule?   I guess we always build from clean anyway, so its not like you could build and then check later with the current setup?

Reply:
The check code is very quick -- it's just a diff of a few files -- so we can just always do the check as part of the regular build.